test/qunit/tests/list-of-pairs-collection-creator.js

```diff
@@ -16,6 +16,31 @@ define([
 
     module( "Galaxy client app tests" );
 
+    test( "Collection creation", function() {
+        var pcc = new PCC({
+                datasets    : DATA._1,
+                historyId   : 'fakeHistoryId'
+            }),
+            server = sinon.fakeServer.create();
+
+        var requestJSON;
+        server.respondWith( 'POST', '/api/histories/fakeHistoryId/contents/dataset_collections', function( request ){
+            requestJSON = JSON.parse( request.requestBody );
+            request.respond(
+                200,
+                { "Content-Type": "application/json" },
+                JSON.stringify({
+                    fakeResponse: 'yes'
+                })
+            );
+        });
+
+        //console.debug( 'requestBody:', JSON.stringify( requestJSON, null, '  ' ) );
+        pcc.createList( 'Heres a collection' );
+        server.respond();
+        deepEqual( requestJSON, DATA._1requestJSON );
+    });
+
```

Comment:
Underscore's `_.uniqueId` function is used to index the datasets in the collection. I noticed that the index is shifted by `1`, when the test is reloaded in the browser e.g. Safari. As a consequence the test only passes every second time. You can try this out by shifting the test back to the end, opening it in the browser and then subsequently reloading the browser window.